circle.yml

```diff
@@ -454,22 +454,37 @@ jobs:
       - build:
           build_type: RelWithDebInfo
           cmake_options: -DFIZZY_FUZZING=ON -DENABLE_ASSERTIONS=ON
+          target: fizzy-fuzz-parser
```

Comment:
What is this doing?

Reply:
It only builds `fizzy-fuzz-parser` instead of building everything, so the build takes only ~15 seconds. We don't run tests in this job (although we could, they would be similar to sanitizers-clang job).

Reply:
It means build target, e.g. `make fizzy-fuzz-parser`?

Reply:
Yes.